fulp_modules/features/antagonists/bloodsuckers/code/clans/clan_brujah.dm

```diff
@@ -0,0 +1,105 @@
+/datum/bloodsucker_clan/brujah
+	name = CLAN_BRUJAH
+	description = "The Brujah seek societal advancement through direct (and usually violent) means.\n\
+		With age they develop a powerful physique and become capable of obliterating almost anything with their bare hands.\n\
+		Be wary, as they are ferverous insurgents, rebels, and anarchists who always attempt to undermine local authorities. \n\
+		Their favorite vassal gains the regular Brawn ability and substantially strengthened fists."
+	clan_objective = /datum/objective/brujah_clan_objective
+	join_icon_state = "brujah"
+	join_description = "Gain an enhanced version of the brawn ability that lets you destroy most structures (including walls!) \
+		Rebel against all authority and attempt to subvert it, but in turn <b>break the Masquerade immediately on joining</b>  \
+		and lose nearly all of your Humanity."
+	blood_drink_type = BLOODSUCKER_DRINK_INHUMANELY
+
+
+/datum/bloodsucker_clan/brujah/New(datum/antagonist/bloodsucker/owner_datum)
+	. = ..()
+	owner_datum.special_vassals.Cut() //Removes Discordant Vassal, which is in the list by default.
```

Comment:
You can just -= the discordant vassal, in case more is added in the future this won't break

Reply:
Done!